content/departments/product-engineering/engineering/cloud/devops/managed/operations.md

```diff
@@ -205,6 +205,32 @@ We are aligned with the [company-wide incident response playbook](../../../proce
 
 We utilize GCP [Uptime Checks](https://cloud.google.com/monitoring/uptime-checks) to perform uptime checks against the [managed instance frontend url](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/f2d46b67f31bfcd2d74f79e46641a701215afb56/modules/terraform-managed-instance/infrastructure.tf#L508-L553). When such alert is fired, it usually means the service is completely not accessible to customers. In the event of downtime, GCP will notify [On-Call DevOps engineers](../index.md#on-call) via Opsgenie and the On-Call engineers will proceed with our incident playbook to ensure we reach to a resolution.
 
+## Confirm instance health
+
+<span class="badge badge-note">SOC2/CI-109</span>
+
+The primary tool that monitors releases post-deployment are through a variety of uptime monitors and system performance metrics. These metrics are covered in documentation related to `SOC/CI-87`.
+
+There is no automatic deployment of Sourcegraph in managed instances. Following a release upgrade, we will perform additional manul check to confirm instance health.
+
+Run command below and inspect the output to ensure that all containers are healthy (in particular, look for anything that says Restarting):
+
+```sh
+mg check
```

Comment:
Although we should have set proper ENVS b/c of creation/upgrade process, how about adding to these commands `[--customer XXX]`? B/c this way you can test any customer, and also this is easier to explain to any auditor.